boards/adafruit/feather_nrf52840/Kconfig.adafruit_feather_nrf52840_sense

```diff
@@ -0,0 +1,7 @@
+# Adafruit Feather nRF52840 Express board configuration
+
+# Copyright (c) 2020 Tobias Svehagen
```

Comment:
The copyrights seem incorrect, this is a file created by Jacub right? copied from a file by Tobias Svehagen
In this case, add your own copyright to the file as well:
```
# Copyright (c) 2020 Tobias Svehagen
# Copyright (c) 2024 Jakub W
```

Reply:
Yes that's correct. Ok I can amend as requested,

What's the threshold for adding a copyright? I made at least minor modifications to most of the files (i.e. in addition to renaming them). Should I amend all files I edited with a copyright?

Reply:
Don't know exactly :) I just want to be on the safe side